src/elements/content-sidebar/activity-feed/annotations/AnnotationActivity.scss

```diff
@@ -46,5 +46,9 @@
     .bcs-AnnotationActivity-link {
         color: $bdl-box-blue;
         font-weight: bold;
+
+        &[aria-disabled='true'] {
+            color: $bdl-gray-62;
```

Comment:
Do we need to add any focus/hover rules in here? Buttons can be really picky.

Reply:
Design said disabled button that is focusable is 👌